boards/uvify/core/init/rc.board_extras

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# UVify UVF4 specific board extras init
+#------------------------------------------------------------------------------
+# IFO
+if param compare SYS_AUTOSTART 4053
+then
+   # Wifi module : 1st telemetry
+   mavlink start -d /dev/ttyS1 -b 921600 -m onboard
+   # Sub 1-Ghz : 2nd telemetry
+   mavlink start -d /dev/ttyS2 -b 57600 -m onboard
+
+   # Change rate to 400 Khz for fast barometer
+   fmu i2c 1 400000
+   
+   # IFO has only external i2c barometer.
+   # It does not start EKF2 in the beginning which is strange behaviour. but 3 seconds hack.
+   # We intentionally put this initialization to here for delayed initialization.
+   sleep 4
+   ms5611 -T 0 -X start
+fi
+
+# Draco
+if param compare SYS_AUTOSTART 4054
```

Comment:
This can be moved to `ROMFS/px4fmu_common/init.d/airframes/4054_draco`.

Reply:
This is just in case someone set wrong autostart number, then have problem.
What do you think?